google/cloud/firestore_v1/async_batch.py

```diff
@@ -30,10 +31,10 @@ class AsyncWriteBatch(BaseWriteBatch):
             The client that created this batch.
     """
 
-    def __init__(self, client):
+    def __init__(self, client) -> None:
         super(AsyncWriteBatch, self).__init__(client=client)
 
-    async def commit(self):
+    async def commit(self) -> Union[Coroutine[Any, Any, list], list]:
```

Comment:
I'm a bit lost - why is the `Union` necessary?

Reply:
This should have been removed. it turns out the actual type here is list. the coroutine bit was added errantly by pytype generation.